src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -166,9 +166,10 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
     })
     Promise.all(promises).then(profiles => {
       const profileAndAccessType:Accessor[] = profiles.map((item, i) => {
+        const accessType = accessorChanges && accessorChanges[i]?.type ? accessorChanges[i].type : AccessType.GAIN_ACCESS
         return {
           profile: item,
-          accessType: accessorChanges[i].type
```

Comment:
This is the source of the bug. `accessorChanges` is undefined when a user creates a new AR. And the app just throws an error and the user is not added.